### PR TITLE
ADEN-5518 Fix localStorage on Safari private mode

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: resources/wikia/modules/localStorage.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ define('wikia.localStorage', ['wikia.window'], function(window) {
 	'use strict';
 
 	try {
+		window.localStorage.setItem('localStorageTestItem', 'testValue');
+		window.localStorage.getItem('localStorageTestItem');
```
Comment (themech) [inline resources/wikia/modules/localStorage.js]: Just wondering: how does this module differ from https://github.com/Wikia/app/blob/b03df0a89ed672697e9c130d529bf1eb25f49cda/resources/wikia/modules/cache.js#L39, which makes basically the same check?

Comment (fraszczakszymon) [reply resources/wikia/modules/localStorage.js]: Fair point. It's used by one module, so I'm going to refactor it a little bit and remove `wikia.localStorage`. Thanks!

Comment (themech) [reply resources/wikia/modules/localStorage.js]: Thank you @fraszczakszymon

**Comment by dianafa** [comment]
+4 −28 and a bug fix, love that! <3

**Pull request merged**